[2021-05-08 18:40:22,993] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.move_files 2021-05-07T00:00:00+00:00 [queued]>
[2021-05-08 18:40:23,013] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.move_files 2021-05-07T00:00:00+00:00 [queued]>
[2021-05-08 18:40:23,013] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-08 18:40:23,013] {taskinstance.py:1069} INFO - Starting attempt 8 of 10
[2021-05-08 18:40:23,013] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-08 18:40:23,133] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): move_files> on 2021-05-07T00:00:00+00:00
[2021-05-08 18:40:23,138] {standard_task_runner.py:52} INFO - Started process 192802 to run task
[2021-05-08 18:40:23,144] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_4', 'move_files', '2021-05-07T00:00:00+00:00', '--job-id', '82', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/dags/dag__4.py', '--cfg-path', '/tmp/tmpt5lgwnkq', '--error-file', '/tmp/tmp458mua90']
[2021-05-08 18:40:23,145] {standard_task_runner.py:77} INFO - Job 82: Subtask move_files
[2021-05-08 18:40:23,313] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_4.move_files 2021-05-07T00:00:00+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-08 18:40:23,350] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=navneet@metaorigins.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag_4
AIRFLOW_CTX_TASK_ID=move_files
AIRFLOW_CTX_EXECUTION_DATE=2021-05-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-07T00:00:00+00:00
[2021-05-08 18:40:23,351] {python.py:118} INFO - Done. Returned value was: None
[2021-05-08 18:40:23,358] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=dag_4, task_id=move_files, execution_date=20210507T000000, start_date=20210508T131022, end_date=20210508T131023
[2021-05-08 18:40:23,701] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-08 18:40:23,715] {local_task_job.py:146} INFO - Task exited with return code 0
